Content.Server/Atmos/Monitor/Systems/AirAlarmSystem.cs

```diff
@@ -188,7 +188,7 @@ private void OnClose(EntityUid uid, AirAlarmComponent component, BoundUIClosedEv
 
         private void OnInteract(EntityUid uid, AirAlarmComponent component, InteractHandEvent args)
         {
-            if (!_interactionSystem.InRangeUnobstructed(args.User, args.Target))
+            if (!_interactionSystem.InRangeUnobstructed(args.User, args.Target, 2, Shared.Physics.CollisionGroup.Impassable, null, true))
```

Comment:
If all you want to do is specify the ignoreInsideBlocker, you don't have to override all the other defaults. you can (and should) just do:
```suggestion
            if (!_interactionSystem.InRangeUnobstructed(args.User, args.Target, ignoreInsideBlocker: true))
```
and similarly in fire alarm system

Reply:
Good to know.